x/docs/handler.go

```diff
@@ -3,7 +3,7 @@ package docs
 import (
 	"fmt"
 
-	"github.com/commercionetwork/commercionetwork/types"
+	types2 "github.com/commercionetwork/commercionetwork/x/docs/internal/types"
```

Comment:
I think we can remove this renaming

```suggestion
	"github.com/commercionetwork/commercionetwork/x/docs/internal/types"
```